srcpkgs/cloog/template

```diff
@@ -10,8 +10,8 @@ short_desc="Library that generates loops for scanning polyhedra"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://www.bastoul.net/cloog/"
-distfiles="http://www.bastoul.net/cloog/pages/download/cloog-$version.tar.gz"
-checksum=325adf3710ce2229b7eeb9e84d3b539556d093ae860027185e7af8a8b00a750e
+distfiles="https://github.com/periscop/cloog/releases/download/cloog-${version}/cloog-${version}.tar.gz"
```

Comment:
What info do we have on this being an "official" fork?

Reply:
The contributors include the original author and most of the people who are also active with isl and cloog development and on cloog-development googlegroup.